### PR TITLE
(motor t)ea time

### PR DESCRIPTION
**Pull request by LynxSolstice**
# About the pull request

Re-adds VCs, changes the VC bay to be a motor T pool for two trucks. Renames the van and it's modules. Removes the van spawns from the groundside maps. Tweaks the speed of the truck. Increases the amount of Marines the trucks can support. (See the changelog for additional minor details.)

# Explain why it's good for the game

The reason the Vehicle Crewman role was removed in the first place is because it was used by players to utilize the APC (a fundamentally flawed vehicle) to frontline and body block. The changes needed to re-add the APC would reduce it to nothing more than a van. (Removal of weapons/frontlining capabilities, reducing it to a 2x3 tile vehicle.) This PR cuts out the weeks of painful tweaking of the APC, (which would end with a reskinned van either way) and adds two vans to the VC bay, removes the ASRS, and allows vehicle gameplay to return to CM in a purely support role. This support role will greatly increase the logistical component of CM's gameplay loop. (Supplies and men can be ferried from the FOB to the front and vise-versa on a rotating basis using the two trucks.)

# Testing Photographs and Procedure

https://youtu.be/UbQwU--C2to

# Changelog

:cl:
add: Vehicle Crewman role un-deleted.
balance: Reduces the speed of the Truck from .5 to .8, bringing it roughly in-line with the APC. Increased the maximum allowed Marines inside to 11, and the maximum allowed xenos to 3. Removed overdrive to keep the speed consistent, (the speed of overdrive was unbalanced.) Removed truck wheels from the colonies. Changed class of truck to medium, in order to allow it to haul large crates, does not effect health or speed.
maptweak: Removes the vehicle ASRS and modifies the VC bay to accommodate two trucks. Modified the interior of the Truck to use APC seats, removed co-driver seat, made room at the back for cargo. Re-adds the Vehicle Crewman role and landmarks for spawning. Removed the truck wheels from spawning on the colonies (RIP surv van shenanigans.) Changed van.dmm to an entirely new map, in-order to expand the interior space from 2x4 to 2x5.
spellcheck: Changed name of the van to USCM Utility Truck. Changed description of the van to reflect it's new status as a truck.
/:cl:

### CODE REVIEW

File: code/modules/vehicles/hardpoints/wheels/van_wheels.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /obj/item/hardpoint/locomotion/van_wheels
-	name = "Van wheels"
-	desc = "Integral to the movement of the van"
+	name = "Truck wheels"
```
Comment (harryob) [inline code/modules/vehicles/hardpoints/wheels/van_wheels.dm:2]:
```suggestion
	name = "truck wheels"
```

**Review by harryob — CHANGES_REQUESTED** [review]

**Review by harryob — APPROVED** [review]

**Review by Agameofscones — CHANGES_REQUESTED** [review]
Removal of chair on LV and spare tires on Trijent intentional? Should restore them. Namely the tires, the chair is w/e

image

Should use these stairs that have the cap on both east and west. The icon state you want is `"p_stair_sn_solo_cap"`

**Review by harryob — CHANGES_REQUESTED** [review]
address ci

**Review by morrowwolf — COMMENTED** [review]
I appreciate the work you've put into this but there's only two ways vehicles are coming back for the foreseeable future:

Req van that is explicitly ran by cargo techs along with the removal of droppodding supplies.

Nerfed APC that disincentivizes frontlining.

**Pull request closed without merging**